.github/scripts/get_benchmark_summary.py

```diff
@@ -0,0 +1,76 @@
+import glob
+import os
+
+import fire
+import pandas as pd
+
+
+def generate_report(report_path: str):
```

Comment:
could just merge the csvfiles and use add_summary to gen summary
https://github.com/InternLM/lmdeploy/blob/4b2c946776a37cce8f9936b639b9999156a9a101/.github/scripts/action_tools.py#L60

Reply:
better put the function in https://github.com/InternLM/lmdeploy/blob/main/.github/scripts/action_tools.py